### ISSUE TITLE
API Development

### ISSUE THREAD

**Issue by chappoo** [XeroAPI/Xero-Net#29]
I noticed there have been no commits to this repository since 05/09/2014:  https://github.com/XeroAPI/Xero-Net/commit/e338a31afee23b788d18ba8250ca747532d05360 however there have been [6 minor releases](http://developer.xero.com/) of the XERO API since then.  Is this repository still under active development?

Happy to get involved in helping to maintain the API (we're already using the .NET wrapper), need to know up front though if the effort is worth it.

**Comment by ronanq** [comment]
Hi folks

Thanks for your feedback.

For the avoidance of doubt, there is only one officially supported .NET wrapper and this is it.
It is quite fair that progress in the latter part of 2014 was erratic - it was a bit of a chicken and egg situation waiting to see what else was needed, but @MatthewSteeples's position reflects a number of Xero API consumers who are not motivated to change, given this involves a fair investment of effort.

We have recently invested a good deal more time bringing this repository up to scratch, and supported an additional API (Files). 

We are also working through outstanding pull requests and looking at automating some of the processes of accepting these as there have been a few that solved a specific developer's issue, but caused problems elsewhere.

The bottom line from us is that we are determined to make this work :)

I will close this specific issue as I hope recent work and my update above resolves it, but we welcome more feedback as we go.

**Comment by chappoo** [comment]
Thanks Ronan; appreciate the update.
